cedar-policy/src/api.rs

```diff
@@ -904,6 +904,11 @@ impl Diagnostics {
     pub fn errors(&self) -> impl Iterator<Item = &AuthorizationError> + '_ {
         self.errors.iter()
     }
+
+    /// Get the `PolicyId`s of the policies where an error occurred during authorization.
+    pub fn error_ids(&self) -> impl Iterator<Item = &PolicyId> {
```

Comment:
Maybe a more explicit name like `error_policy_ids`. `error_ids` makes it sound like they might be error code.

Reply:
Fair point. Changed in the latest commit.